addons/project_timesheet_holidays/tests/test_timesheet_global_time_off.py

```diff
@@ -271,3 +271,59 @@ def test_timesheet_creation_and_deletion_for_calendar_update(self):
         self.assertIn(gto_09_04, global_leaves_ids_35h)
         self.assertIn(gto_11_13, global_leaves_ids_35h)
         self.assertNotIn(gto_09_11, global_leaves_ids_35h)
+
+    def test_global_time_off_timesheet_creation_after_leave_refusal(self):
```

Comment:
I know we did not do in 15.2, but we could also add a small test to check the expected behavior when the user cancels one leave. 🙂

Reply:
Added tests for cancelling the leave. Not sure if this is the best/cleanest way to do this since some code is repeated. I thought about creating a sub function but it seemed unnecessary for just 2 cases.

Reply:
No problem, it is good to me. 🙂